completions/functions.py

```diff
@@ -111,7 +111,6 @@
     "fit-content": t.length + t.percentage,
     "format": [t.string],
     "frames": t.integer,
-    "gray": t.number + t.percentage,
```

Comment:
This reference might not actually be problematic in terms of autocompletions, but it should be removed regardless since the `gray()` function no longer exists